GTViewController/Classes/Custom Classes/UISnuffleButton.m

```diff
@@ -182,7 +182,7 @@ - (id)createButtonFromElement:(TBXMLElement *)element addTag:(NSInteger)tag yPos
             }
             contentHorizontalAlignment	= UIControlContentHorizontalAlignmentCenter;
             contentVerticalAlignment	= UIControlContentVerticalAlignmentCenter;
-        } else if ([mode isEqual:kButtonMode_link]) {
+        } else if ([mode isEqual:kButtonMode_link] || [[TBXML elementName:element] isEqual:@"link-button"]) {
```

Comment:
You should probably use a constant for @"link-button" instead of the raw value so it is easier for people to come to the top of the file and check the spec.